test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/UserReferenceRequest.java

```diff
@@ -32,20 +32,27 @@
      * @param client         the service client
      * @param requestOptions the options for this request
      */
-    public UserReferenceRequest(@Nonnull final String requestUrl, @Nonnull final IBaseClient client, @Nullable final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions) {
+    public UserReferenceRequest(String requestUrl, IBaseClient client, java.util.List<? extends com.microsoft.graph.options.Option> requestOptions) {
```

Comment:
As the Nonnull annotation is removed, are null parameters allowed and if yes, are they handled by the code?

Reply:
sorry about that, base rebase. Fixed